.github/workflows/clean-tsv.yml

```diff
@@ -0,0 +1,45 @@
+name: Tab correcting and sorting on manually maintained tsv files
+
+on:
+  push:
+    branches:
+      - clean-metadata
+    tags-ignore:
+      - '**'
+    paths:
+      - source-data/location_hierarchy.tsv
+      - source-data/gisaid_annotations.tsv
+
```

Comment:
I know you mentioned changing this `branches:` value to "master" before merging.
Just to be clear, you want this workflow to run on every new push to master?

Reply:
Yes, every push with changes to either of these files:
```
     paths:
      - source-data/location_hierarchy.tsv
      - source-data/gisaid_annotations.tsv
```
similar to https://github.com/nextstrain/ncov-ingest/blob/master/.github/workflows/ingest-gisaid-master.yml#L5-L10